api/alembic/versions/0d46262707af_alter_morecast_forecast_columns_to_float.py

```diff
@@ -0,0 +1,34 @@
+"""Alter morecast_forecast columns to float
+
+Revision ID: 0d46262707af
+Revises: 69cbd7ca2477
+Create Date: 2023-03-14 15:55:31.325373
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '0d46262707af'
+down_revision = '69cbd7ca2477'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.drop_column('morecast_forecast', 'temp')
+    op.drop_column('morecast_forecast', 'precip')
+    op.drop_column('morecast_forecast', 'wind_speed')
+    op.add_column('morecast_forecast', sa.Column('temp', sa.Float(), nullable=False))
+    op.add_column('morecast_forecast', sa.Column('precip', sa.Float(), nullable=False))
+    op.add_column('morecast_forecast', sa.Column('wind_speed', sa.Float(), nullable=False))
```

Comment:
I think we should also store the `create_timestamp` here and use that in the response. Similar to how the `planning_weather_stations` table does.

Reply:
The create_timestamp field already exists on the model and is returned in the response to the POST. Check out the /forecast route (line 48 of app/routers/morecast_v2.py).

Reply:
Whoops my bad

Reply:
I'm glad it's my friday... :)